sematic/ui/packages/common/src/component/GitInfo.tsx

```diff
@@ -36,12 +42,12 @@ const GitInfoBox = () => {
             <Link>pf49df3</Link>
             <CopyButton text={"pf49df3"} />
         </StyledBox>
-        <StyledBox>
+        {!!hasUncommittedChanges && <StyledBox>
             <Code />
             <Tooltip title={"This run used code with uncommitted changed on top of the above commit."} arrow={true} >
                 <Typography>Uncommited changes</Typography>
```

Comment:
Are we intentionally replacing the existing icon with this?
![image](https://user-images.githubusercontent.com/1894533/228914245-2c3d686c-4125-4941-aea8-090478d83210.png)